src/core/issues-notes.js

```diff
@@ -99,26 +99,19 @@ const l10n = getIntlData(localizationStrings);
  * @param {*} conf
  */
 function handleIssues(ins, ghIssues, conf) {
-  const hasDataNum = !!document.querySelector(".issue[data-number]");
-  let issueNum = 0;
+  const getIssueNumber = createIssueNumberGetter();
   const issueList = document.createElement("ul");
   ins.forEach(inno => {
     const { type, displayType, isFeatureAtRisk } = getIssueType(inno);
     const isIssue = type === "issue";
     const isInline = inno.localName === "span";
     const { number: dataNum } = inno.dataset;
-    /** @type {Partial<Report>} */
     const report = {
       type,
       inline: isInline,
       title: inno.title,
+      number: getIssueNumber(inno),
```

Comment:
is this assured not to throw? createIssueNumberGetter can return undefined.

Reply:
Yes, tests assure that numberless issues silently gets no number, by checking `report.number !== undefined`.